site/content/docs/developing/additional-aws-resources.ja.md

```diff
@@ -117,3 +117,36 @@ Outputs:
     
     * `addons/` ディレクトリで定義するポリシーでは[最小限のアクセス権を付与する](https://docs.aws.amazon.com/ja_jp/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
     * `addons/` ディレクトリで定義したリソースに対してのみアクセスできるようにポリシーを制限するために [追加セキュリティに対するポリシー条件を使用する](https://docs.aws.amazon.com/ja_jp/IAM/latest/UserGuide/best-practices.html#use-policy-conditions) 
+
+
+### `Parameters` セクションのカスタマイズ
+
+Copilot では、 `App`, `Env` そして `Name` [パラメーター](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)がテンプレートに定義されている必要があります。もし、 Service スタック内のリソースを参照する[パラメーター](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)を追加したい場合は、 `addons.parameters.yml` ファイルを作成します。
+
+```term
+.
+└── addons/
+    ├── template.yml
+    └── addons.parameters.yml # このファイルをaddons/ディレクトリの下に追加します
```

Comment:
```suggestion
    └── addons.parameters.yml # このファイルを addons/ ディレクトリの下に追加します
```

Reply:
Thank you. I overlooked it. 🙇️  
(including other parts).